portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,13 +46,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
```

Comment:
Consider expanding your commit messages.

"Improve website interface for better user experience"

Consider something like:

Improve website interface for better user experience

* Simplified user name input
* Added mail implementation that allows users to send me an email via....
* etc

This is useful as you start moving faster and want to go back and see changes you made in the past.

Reply:
Note - you don't need to change anything here for this PR. This is just a recommendation for future PRs.